doc/mgr/orchestrator.rst

```diff
@@ -115,6 +115,35 @@ Service Commands::
 
     ceph orch <start|stop|restart|redeploy|reconfig> <service_name>
 
+   .. note:: these commands applies to cephadm containerized daemons only.
+
+Options
+=======
+
+.. option:: start
+
+   Start the daemon on the corresponding host.
+
+.. option:: stop
+
+   Stop the daemon on the corresponding host.
+
+.. option:: restart
+
+   Restart the daemon on the corresponding host.
+
+.. option:: redeploy
+
+   Redeploy the ceph daemon on the corresponding host. This will recreate the daemon directory
+   structure under ``/var/lib/ceph/<fsid>/<daemon-name>`` (if it doesn't exist), refresh its
+   configuration files, regenerate its unit-files and restarts the systemd daemon.
+
+.. option:: reconfig
+
+   Reconfigure the daemon on the corresponding host. This will refresh the ceph daemon configuration
+   files and restarts its systemd daemon.
+
```

Comment:
As above, I want to be sure that we're both accurate and precise, to avoid confusing operators (like me).

Is it true that there will only be a systemd daemon (unit? target?) when the daemon is containerized with Podman, not with Docker?  Do these commands apply to Dockerized daemons?

If so, maybe something more like the below is accurate and covers both?

`Reconfigure the daemon on the corresponding host. This will refresh configuration files then restart the daemon.`